### ISSUE TITLE
Interface provides no feedback when exemplar image changed

### ISSUE THREAD

**Issue by BobCorrigan** [EOL/tramea#191]
On this taxon page http://www.eol.org/pages/1056566/media I switched the exemplar photo from one image to another by clicking on the radio button next to "set as exemplar".  The interface provided no feedback.  When I went to check if the new photo was set as the exemplar by changing to the taxon overview page, the photo had changed.

**Comment by KatjaSchulz** [comment]
I think this should be in the "get it right in Tramea" category.  Rating and setting exemplars was always buggy in V2.  Hopefully, we can set it up differently in V3, so things don't break every couple of weeks.

**Comment by jhammock** [comment]
Agreed; I believe this is the behavior we have almost always, if not always had. The action is recorded on the overview tab in the curator activity feed, for what that's worth.

**Comment by BobCorrigan** [comment]
For some reason I recalled a notice on the screen but no biggie. I logged it because I thought that function vanished and might be an indicator of something else going on. Can be ignored.

**Comment by JRice** [comment]
Yeah, I can confirm we have JS that is _supposed_ to update the image "in real time" when you select an exemplar image. It's broken. But if we don't care, I don't care.  :D
